VictoriaAlbert/VictoriaAlbert/APIRequestManager.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  APIRequestManager.swift
+//  VictoriaAlbert
+//
+//  Created by Karen Fuentes on 11/10/16.
+//  Copyright © 2016 Karen Fuentes. All rights reserved.
+//
+
+import Foundation
+
+class APIRequestManager {
+    
+    static let manager = APIRequestManager()
+    private init() {}
+    
+    func getData(endPoint: String, callback: @escaping (Data?) -> Void) {
+        guard let myURL = URL(string: endPoint) else { return }
+        let session = URLSession(configuration: URLSessionConfiguration.default)
+        session.dataTask(with: myURL) { (data: Data?, response: URLResponse?, error: Error?) in
+            if error != nil {
+                print("Error durring session: \(error)")
+            }
+            guard let validData = data else { return }
+            callback(validData)
+            }.resume()
+    }
+    
+    func getImage(urlString: String, callback: @escaping (Data) -> () ) {
+        
```

Comment:
Notice how getImage and getData are identical but for some naming. There's really only need for one.